### ISSUE TITLE
Create notification email template

### ISSUE THREAD

**Issue by thebitguru** [CV19Assist/app#50]
One of the immediate needs is to send out notification emails to volunteers for open requests.  We are still analyzing the initial approach as part of #49, but our initial vision is to check every 5 hours and send a notification email to volunteers for open requests.  The notification email will include the following sections.

1. Open requests near you: this section will include up to 15 open needs within a 60 mile radius, ordered by the distance from the individual volunteer.

2. Open requests farther away: this section will include up to 5 open needs that are farther away, ordered by the distance from the volunteer.  The intention here is to request help with either finding more volunteers or organizations who might be able to assist.

3. Simple informational text about the upcoming feature to snooze or turn off these notification emails.

**Comment by thebitguru** [comment]
@melissa-easker thank you very much for your work on #48!  I will be getting back on that soon.  Any chance you can also help with this request?

**Comment by melissa-easker** [comment]
Sure no problem. If I can get access to your MailChimp, I can set up something directly on the platform along with converting #48 to a responsive html template. Let me know if that's a possibility.

**Comment by thebitguru** [comment]
Sure, I just invited you to our Mailchimp account.  In this case, though, please note that the email will be sent by the application code and not through Mailchimp, so the template will have to be converted to HTML at some point.  Thanks!

**Comment by melissa-easker** [comment]
Oh well in that case, I can just create it straight in HTML.

**Comment by melissa-easker** [comment]
The emails sent through the email services like mail chimp uses templates
thaf are already html BTW.

**Comment by thebitguru** [comment]
Good point about the templates already being HTML.  Given this, I guess it doesn't matter where you end up composing them.

**Comment by melissa-easker** [comment]
Sounds good. I converted the volunteer update to HTML last night/this
morning and will move on to the notification email directly in MailChimp
along with a "How it Works" process graphic over the weekend.

**Comment by melissa-easker** [comment]
![Request Update](https://user-images.githubusercontent.com/53533052/79046228-4ab75200-7bd5-11ea-9ef9-4720aa5055cb.png)
Template created. Right now the images are all static pulled from the website but it would be neat if this could be automated somehow. I've shown the map, requestor name, and approximate location as well as the submission date. This keeps the more sensitive information hidden until the person clicks through to the page. I've used messaging that @saadbaig2020  provided in his email.

**Comment by thebitguru** [comment]
Thanks, @melissa-easker!  We will be working on the emails in the coming days and will follow up.

**Comment by saadbaig2020** [comment]
this is really great. thanks for putting it together, @melissa-easker! I think we can remove the "This is Saad..." paragraph and replace it with something like "Hi *volunteer*, we found 2 new requests new you. Click "more details" to view and pick up a request if you're able to fulfill it."

**Comment by thebitguru** [comment]
@saadbaig2020, I just added you to the Mailchimp account.  Let's edit the content in the template in Mailchimp which then @prescottprue and I can move it to the frontend where it will finally live.

**Comment by saadbaig2020** [comment]
is the template called Test in Mailchimp?

**Comment by prescottprue** [comment]
@thebitguru sounds good. In [this PR](https://github.com/CV19Assist/app/pull/65) I moved [the templates into the repo](https://github.com/CV19Assist/app/pull/65/files#diff-6bf879c57c015e15e8bd178d447721de) and made it so their [content is written to Firestore when on deploy](https://github.com/CV19Assist/app/pull/65/files#diff-3680803a67ea02b9d6b8b783f532a8ceR104) - we can update them from mailchimp manually for now, but there may be a way to wire the two together
